client/metadata/assets/mainnet.ini

```diff
@@ -269,91 +431,204 @@ min_quantity_tick_size = 100000
-decimals = 18
+[0x8c7fd5e6a7f49d840512a43d95389a78e60ebaf0cde1af86b26a785eb23b3be5]
+description = 'Mainnet Derivative OSMO/UST PERP'
+base = 0
+quote = 6
+min_price_tick_size = 1000
+min_display_price_tick_size = 0.001
+min_quantity_tick_size = 0.01
+min_display_quantity_tick_size = 0.01
 
-[MATIC]
-peggy_denom = peggy0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0
-decimals = 18
+[0xcf18525b53e54ad7d27477426ade06d69d8d56d2f3bf35fe5ce2ad9eb97c2fbc]
+description = 'Mainnet Derivative OSMO/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 1000
+min_display_price_tick_size = 0.001
+min_quantity_tick_size = 0.01
+min_display_quantity_tick_size = 0.01
 
-[GRT]
-peggy_denom = peggy0xc944E90C64B2c07662A292be6244BDf05Cda44a7
+[0x06c5a306492ddc2b8dc56969766959163287ed68a6b59baa2f42330dda0aebe0]
+description = 'Mainnet Derivative SOL/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 10000
+min_display_price_tick_size = 0.01
+min_quantity_tick_size = 0.1
+min_display_quantity_tick_size = 0.1
+
+[0x3b7fb1d9351f7fa2e6e0e5a11b3639ee5e0486c33a6a74f629c3fc3c3043efd5]
+description = 'Mainnet Derivative BONK/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 0.0001
+min_display_price_tick_size = 1e-10
+min_quantity_tick_size = 0.1
+min_display_quantity_tick_size = 0.1
+
+[0xcd0c859a99f26bb3530e21890937ed77d20754aa7825a599c71710514fc125ef]
+description = 'Mainnet Derivative 1MPEPE/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 100
+min_display_price_tick_size = 0.0001
+min_quantity_tick_size = 1
+min_display_quantity_tick_size = 1
+
+[0x63bafbeee644b6606afb8476dd378fba35d516c7081d6045145790af963545aa]
+description = 'Mainnet Derivative XRP/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 100
+min_display_price_tick_size = 0.0001
+min_quantity_tick_size = 0.1
+min_display_quantity_tick_size = 0.1
+
+[0x1afa358349b140e49441b6e68529578c7d2f27f06e18ef874f428457c0aaeb8b]
+description = 'Mainnet Derivative SEI/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 100
+min_display_price_tick_size = 0.0001
+min_quantity_tick_size = 0.1
+min_display_quantity_tick_size = 0.1
+
+[0x4b7a550e1760cbd2c6ccb97562c56878083b92e32d07e9e3326945c6e60e6b60]
+description = 'Mainnet Derivative TIA/USDT-30NOV2023 PERP'
+base = 0
+quote = 6
+min_price_tick_size = 1000
+min_display_price_tick_size = 0.0001
+min_quantity_tick_size = 0.1
+min_display_quantity_tick_size = 0.1
+
+[0x4fe7aff4dd27be7cbb924336e7fe2d160387bb1750811cf165ce58d4c612aebb]
+description = 'Mainnet Derivative AXL/USDT PERP'
+base = 0
+quote = 6
+min_price_tick_size = 100
+min_display_price_tick_size = 0.0001
+min_quantity_tick_size = 1
+min_display_quantity_tick_size = 0.1
+
+[0x332230109e7afb839b4750d4cf961666b608071ecb64dac55662dac37529639e]
+description = 'Mainnet Derivative BTC/USDTkv PERP'
+base = 0
+quote = 6
+min_price_tick_size = 1000000
+min_display_price_tick_size = 0.0001
+min_quantity_tick_size = 0.0001
+min_display_quantity_tick_size = 0.1
```

Comment:
`min_display_quantity_tick_size = 0.0001` (for perp min_quantity_tick_size = min_display_quantity_tick_size)